src/components/Footer/Footer.jsx

```diff
@@ -3,15 +3,18 @@ import "../Footer/Footer.css";
 
 function Footer() {
   return (
-    <div className="footer w-full p-5 flex flex-col items-center bg-indigo-500">
-      <h2 className="font-bold  -mt-3">Follow Us </h2>
-      <div className=" m-auto flex flex-row justify-between items-center w-1/6  mt-3  ">
-        <img src="https://img.icons8.com/ios-filled/30/null/facebook-new.png" />
+    <div className="footer w-full p-2 mt-28 flex flex-col items-center bg-red-500">
+      <h2 className="font-bold  ">Follow Us on </h2>
+      <div className=" flex flex-row justify-between h-10 items-center w-1/6   ">
+        <img
+          m-0
```

Comment:
className='m-0'?

Reply:
Overlooked this and even it didn't threw an error too, resolved it now.
Thanks for pointing it out 👍🏻